templates/partial/_breadcrumbs.html

```diff
@@ -0,0 +1,16 @@
+<div class="row">
```

Comment:
We have a partial already [here](https://github.com/canonical/ubuntu.com/blob/main/templates/shared/_breadcrumbs.html) so I think this can be removed